docs/src/main/docs/guides/07_datasource.adoc

```diff
@@ -343,3 +348,19 @@ https://docs.oracle.com/en/database/oracle/oracle-database/19/jjuar/oracle/ucp/j
 implementation instance whose name is specified by the `@Named`
 annotation, and will set this field's value to it.
 
+== Examples
+
+Helidon features a few examples of projects that use data sources.
+
+* https://github.com/oracle/helidon/tree/master/examples/integrations/cdi/datasource-hikaricp-h2[An
```

Comment:
Can you point at examples in the release tag using the project-version attribute instead ?

Reply:
Oh, that's a good idea.